aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/SdkConfigDecorator.kt

```diff
@@ -78,6 +78,7 @@ class GenericSmithySdkConfigSettings : ClientCodegenDecorator {
 
                     ${section.serviceConfigBuilder}.set_http_connector(${section.sdkConfig}.http_connector().cloned());
 
+                    ${section.sdkConfig}.interceptors().cloned().for_each(|interceptor| {${section.serviceConfigBuilder}.set_interceptor(interceptor);});
```

Comment:
If `set_interceptors` is added, then this can just be a single function call rather than a loop.

Reply:
Addressed in ba26b98.